server/db.js

```diff
@@ -1,11 +1,12 @@
 import { MongoClient } from "mongodb";
 import * as dotenv from "dotenv";
-
 dotenv.config();
 
-const dbUrl = process.env.MONGODB_URI || "mongodb://localhost:27017/cyf";
-const configuration = { useNewUrlParser: true, useUnifiedTopology: true };
-
-export const getClient = () => new MongoClient(dbUrl, configuration);
```

Comment:
Although it is not important, we can discuss this to learn to correct one. To me it is URL. But no impact on the code.

Reply:
I think they are used interchangeably, in MongoDB documentation I saw both URL and URI, I just used URI because it is more general. But I don't mind changing it.

Reply:
There is a difference but small one. I just wanted to let us know about it. I posted an article on the channel. Have a look at it to get some knowledge. Otherwise both works here.